minigalaxy/launcher.py

```diff
@@ -88,9 +102,25 @@ def __get_execute_command(game) -> list:
             return ["scummvm", "-c", scummvm_config]
 
     # Wine
-    if "prefix" in files and shutil.which("wine"):
-        # This still needs to be implemented
-        return [os.path.join(game.install_dir, "start.sh")]
+    if shutil.which("wine"):
```

Comment:
This check needs to change to something filename related or it should move somewhere else, either at the bottom of the method or in a different method